mailbox/jpa/src/main/java/org/apache/james/mailbox/jpa/mail/model/JPAAttachment.java

```diff
@@ -0,0 +1,205 @@
+import org.checkerframework.checker.units.qual.C;
+
+import javax.persistence.*;
+import java.util.*;
+
+@Entity(name = "Attachment")
+@Table(name = "JAMES_MAIL_ATTACHMENT")
+public class JPAAttachment {
+
+    public static final String ATTACHMENT_ID = "ATTACHMENT_ID";
+    public static final String ATTACHMENT_CONTENT = "CONTENT";
+    public static final String ATTACHMENT_TYPE = "TYPE";
+    public static final String ATTACHMENT_OWNERS = "OWNERS";
+    public static final String ATTACHMENT_MESSAGE_IDS = "MESSAGE_IDS";
+    public static final String ATTACHMENT_CID = "CID";
+    public static final String ATTACHMENT_IS_INLINE = "INLINE";
+
+
+    @Id
+    @Column(name = ATTACHMENT_ID, nullable = false, updatable = false)
+    private String attachmentId;
+
+    @Lob
+    @Column(name = ATTACHMENT_CONTENT, columnDefinition = "BLOB NOT NULL")
+    private byte[] content;
+
+    @Column(name = ATTACHMENT_TYPE)
+    private String type;
+
+    @Column(name = ATTACHMENT_OWNERS)
+    @ElementCollection
+    private Collection<String> owners;
+
+    @Column(name = ATTACHMENT_MESSAGE_IDS)
+    @ElementCollection
+    private Collection<String> messageIds;
+
+    @Column(name = ATTACHMENT_CID)
+    private String cidValue;
+
+    @Column(name = ATTACHMENT_IS_INLINE)
+    private Boolean inline;
+
+    public JPAAttachment(){}
+
+    public JPAAttachment(String attachmentId, byte[] content, String type, Collection<String> owners, Collection<String> messageIds, String cid, Boolean inline){
+        this.attachmentId = attachmentId;
+        this.content = content;
+        this.type = type;
+        this.owners = owners;
+        this.messageIds = messageIds;
+        this.cidValue = cid;
+        this.inline = inline;
+    }
+
+    public String getAttachmentId() {
+        return attachmentId;
+    }
+
+    public void setAttachmentId(String attachmentId) {
+        this.attachmentId = attachmentId;
+    }
+
+    public byte[] getContent() {
+        return content;
+    }
+
+    public void setContent(byte[] content) {
+        this.content = content;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public Collection<String> getOwners() {
+        return owners;
+    }
+
+    public void setOwners(Collection<String> owners) {
+        this.owners = owners;
+    }
+
+    public Collection<String> getMessageIds() {
+        return messageIds;
+    }
+
+    public void setMessageIds(Collection<String> messageId) {
+        this.messageIds = messageId;
+    }
+
+    public String getCidValue() {
+        return cidValue;
+    }
+
+    public void setCidValue(String cidValue) {
+        this.cidValue = cidValue;
+    }
+
+    public Boolean getInline() {
+        return inline;
+    }
+
+    public void setInline(Boolean inline) {
+        this.inline = inline;
+    }
+
+
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
I think it need a test with `EqualVerifier`

Reply:
included in the test.